iceoryx_examples/iceperf/iceperf_laurel.cpp

```diff
@@ -71,30 +75,49 @@ void leaderDo(IcePerfBase& ipcTechnology, int64_t numRoundtrips)
 
 int main(int argc, char* argv[])
 {
+    Benchmarks benchmark = Benchmarks::ALL;
```

Comment:
I guess you'll have a problem if laurel and hardy are not started with the same benchmark option...

Reply:
Yes. But you are able to mix the C++ and C API.